lib/codecs/Base58DidUrlDecoder.js

```diff
@@ -4,15 +4,14 @@
 import {CborldDecoder} from './CborldDecoder.js';
 import {encode as encodeBase58} from 'base58-universal';
 
-const ID_TO_SCHEME = new Map([
-  // Note: only v1 mainnet is supported
-  [1024, 'did:v1:nym:'],
-  [1025, 'did:key:']
-]);
-
 export class Base58DidUrlDecoder extends CborldDecoder {
+  constructor({decompressionMap} = {}) {
+    super();
+    this.decompressionMap = decompressionMap;
+  }
```

Comment:
See comment below too...

```suggestion
  constructor({prefix} = {}) {
    super();
    this.prefix = prefix;
  }
```

And you can call this something other than `prefix` if you want (if you've got a better name).